openretina/modules/layers/convolutions.py

```diff
@@ -144,6 +146,10 @@ def __init__(
             padding=padding,
             bias=bias,
         )
+        self.weight_spatial = self.conv.weight.data
+
+    def refresh_spatial_weight_attribute(self):
+        self.weight_spatial = self.conv.weight.data
```

Comment:
```suggestion

    @property
    def weight_spatial(self):
        self.weight_spatial = self.conv.weight.data
```
I think using @property has the same effect and we have less state in this class (e.g. we do not have to think about updating self.weight_spatial at all), see: https://docs.python.org/3/library/functions.html#property